README.md

```diff
@@ -155,6 +155,11 @@ HttpSysBinding has some limitations:
 * The launcher may need to be run as an administrator
 * No Tls certificates can be set
 
+---
+
+**Q:** The Launcher fails with the messages `Host ... has shutdown.` and `Restarting ...`.
```

Comment:
In addition to that we should probably add a known issues / limitations section to document such things. This question is pretty specific to the issue itself, but IMO a section that will keep everybody informed upfront would be helpful